dizoo/metadrive/config/test_ppo.py

```diff
@@ -0,0 +1,107 @@
+
+from easydict import EasyDict
+from functools import partial
+from tensorboardX import SummaryWriter
+
+from ding.envs import BaseEnvManager, SyncSubprocessEnvManager
```

Comment:
polish import sequence accoding to [doc](https://di-engine-docs.readthedocs.io/zh_CN/latest/21_code_style/index_zh.html#import)